src/styles/colorManipulator.spec.js

```diff
@@ -90,25 +90,25 @@ describe('utils/colorManipulator', () => {
     });
   });
 
-  describe('getContrastRatio', () => {
-    it('returns a ratio of 21 for black : white', () => {
```

Comment:
You probably want to keep the word for in these descriptions?

Reply:
Duplicating the assertion in the description can lead to out of date description, e.g. with the last one of this serie.

Reply:
The word "for". 

You have it in some othe later descriptions.

i.e. "it('returns a ratio *for* black : white'."